src/translation.ts

```diff
@@ -26,34 +26,33 @@ function sprintf(s: string, ...values: string[] | [{ [key: string]: string }]):
  * Allow to inject a translation function from outside o-spreadsheet.
  * @param tfn the function that will do the translation
  */
-export function setTranslationMethod(tfn: TranslationFunction) {
+export function setTranslationMethod(tfn: TranslationFunction, loaded: () => boolean = () => true) {
   _translate = tfn;
+  _loaded = loaded;
 }
 
 export const _t: TranslationFunction = function (
   s: string,
   ...values: string[] | [{ [key: string]: string }]
 ) {
+  if (!_loaded()) {
+    return new LazyTranslatedString(s, values) as unknown as string;
+  }
   return sprintf(_translate(s), ...values);
 };
 
-export const _lt: TranslationFunction = function (
-  str: string,
-  ...values: string[] | [{ [key: string]: string }]
-) {
-  // casts the object to unknown then to string to trick typescript into thinking that the object it receives is actually a string
-  // this way it will be typed correctly (behaves like a string) but tests like typeof _lt("whatever") will be object and not string !
-  return new LazyTranslatedString(str, values) as unknown as string;
-};
-
 class LazyTranslatedString extends String {
   constructor(str: string, private values: string[] | [{ [key: string]: string }]) {
     super(str);
   }
 
   valueOf() {
     const str = super.valueOf();
-    return sprintf(_translate(str), ...this.values);
+    if (_loaded()) {
+      return sprintf(_translate(str), ...this.values);
+    } else {
+      throw new Error(`translation error`);
```

Comment:
actually, I don't think it's a good thing to throw an error here. It prevents using the library without setting a translation function.
The setup should be minimal to use the lib and a translation function is definitely an advanced setup